apps/desktop-wallet/src/modals/SettingsModal/WalletsSettingsSection.tsx

```diff
@@ -108,15 +109,40 @@ const WalletsSettingsSection = () => {
             <Button role="secondary" onClick={lockWallet}>
               {t('Lock current wallet')}
             </Button>
-            <Button role="secondary" onClick={() => setIsQRCodeModalVisible(true)}>
-              {t('Export current wallet')}
-            </Button>
+
+            <ButtonTooltipWrapper
+              data-tooltip-id="default"
+              data-tooltip-content={
+                activeWallet.isPassphraseUsed ? t('To export this wallet use it without a passphrase') : ''
```

Comment:
To avoid confusion, passphrase wallets should not be exportable, since we are not exporting the passphrase. The user ends up with a different wallet on their mobile (the one with the 24 words)

Reply:
💯